tests/testthat/helper.R

```diff
@@ -0,0 +1,6 @@
+library(jsonlite)
+
+wipe_results <- function() {
+  unlink(test_path("results/"), recursive = TRUE)
+  unlink(test_path("bm-scripts/"), recursive = TRUE)
```

Comment:
This implies the scripts are part of the results, and should probably be in the same directory, no? Not sure what's ideal here, but maybe an `outputs` dir with `results` and `scripts` subdirs?

Reply:
removed now